stix_shifter_modules/secretserver/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -0,0 +1,91 @@
+{
+  "username": [
+    {
+    "key": "user-account.user_name"
```

Comment:
should be `user-account.display_name`

Reply:
Done the changes

Reply:
https://github.com/opencybersecurityalliance/stix-shifter/pull/1092/commits/0175050f7a72856888ac00aaffe1e64572068624